doc(release) Release notes hotfix-collect 25.10 jun26 (#5381)
centreon/centreon-documentation: versioned_docs/version-25.10/releases/centreon-os.mdx

</details>

## June 8, 2026

### Centreon Collect 25.10.8

<details>
  <summary>Bug fixes</summary>

> - [Configuration] unresolved/empty macros in macros are now ignored unless if surrounded by `{{ }}`.

Reviewer: ```suggestion
- [Configuration] Unresolved/empty macros whithin macros are now ignored unless surrounded by `{{ }}`.
```
